storage/innobase/handler/ha_innodb.cc

```diff
@@ -6789,11 +6797,11 @@ wsrep_store_key_val_for_row(
 			|| mysql_type == MYSQL_TYPE_GEOMETRY) {
 
 			const CHARSET_INFO* cs;
-			ulint		key_len;
-			ulint		true_len;
+			ulint		key_len=0;
+			ulint		true_len=0;
 			int		error=0;
-			ulint		blob_len;
-			const byte*	blob_data;
+			ulint		blob_len=0;
+			const byte*	blob_data=nullptr;
```

Comment:
Instead of adding redundant assignments to some of these local variables, can be please move the declarations to the place of the first assignment, such as the following? (By the way, the type of `key_part->length` seems to be `uint16`.)
```c++
const size_t key_len = key_part->length;
```